schemas/groups/propulsion.json

```diff
@@ -64,6 +41,12 @@
       "units": "Hz"
     },
 
+    "maxRevolutions": {
```

Comment:
Having the dynamic and static data grouped by the entity they are about has advantages and disadvantages.

However I don't see the problem that you mention. If you subscribe to all propulsion values you will not get updates for the static data, as you can get delta updates only for data that has been updated, not the static data.